nnpdf_data/nnpdf_data/filter_utils/legacy_jets_utils.py

```diff
@@ -0,0 +1,1148 @@
+"""
+Utils specifically needed to ease the implementation of the legacy jet datasets,
```

Comment:
why are these "legacy jet dataset".

These are simply the jet datasets for 7 and 8 TeV.